src/ModalHeader.js

```diff
@@ -13,7 +13,7 @@ class ModalHeader extends React.Component {
         { this.props.closeButton &&
           <button
             className='close'
-            aria-label={this.props['aria-label'] || 'Close'} //eslint-disable-line react/prop-types
+            aria-label={this.props['aria-label'] || 'Close'}
             onClick={this.props.onHide}
```

Comment:
sorry one more thought here. since we are explicitly adding aria-label to props, we should also remove `'Close'` here and add it to the defaultProps, so that the docs can report it.